### ISSUE TITLE
Cedar status

### ISSUE THREAD

**Issue by weitzel926** [pivotal/cedar#402]
What is the status of this project?  Is it being maintained?  Is there a plan for maintaining it going into the future?  Thanks!

**Comment by tjarratt** [comment]
Hello @weitzel926 sorry for the delay, let me see if I can explain this.

There have been a lot of folks within Pivotal that have maintained Cedar across the years, including myself. It's OSS and lots of folks have rotated through the codebase - some of them are still with Pivotal and most of the ones that aren't have stopped being active contributors. 

For most of our day-to-day operations our teams have migrated to using Quick and Nimble for writing our unit tests on iOS and Mac OS X, so we don't get as much direct feedback when the Xcode releases break things. 

I'm really passionate about Cedar and plan on investing *some* time this week and next week (focus on *some* since I'm also a parent, manager of a team and interviewing candidates these weeks in between several holidays in France...) but as with all OSS software, we can't promise that we'll have a fix out any given time. In the past I occasionally had a few weeks or month at a time to fix bugs in between my projects at work, but that's becoming increasingly rare lately.

If you're using Cedar for your projects and like using it, I'm happy to help without with fixes and feedback on how to solve problems. 

In short, I'd say "Cedar maintained with love and grace, but we don't have anyone verifying that Xcode and Cedar continue to work well together with each Xcode upgrade. Fixes for bugs will probably be slow coming, but there are a lot of opportunities for contributors to help out if they'd like to."
